src/System.ValueTuple/src/System/ValueTuple/ValueTuple.cs

```diff
@@ -123,7 +138,21 @@ string ITupleInternal.ToStringEnd()
             return ")";
         }
 
-        int ITupleInternal.Size => 0;
+        /// <summary>
+        /// The number of positions in this data structure.
+        /// </summary>
+        public int Length => 0;
```

Comment:
I think these should be explicit implementations. I don't think we want to prevent `Length` from being a valid tuple "friendly" element name.